dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/publish/files/WebServiceRefFeaturesTestServer/image-property.wsdl

```diff
@@ -92,15 +92,15 @@
 		<wsdl:port binding="tns:ImageServiceImplServiceSoapBinding"
 			name="ImageServiceImplPort">
 			<soap:address
-				location="http://localhost:8010/webServiceRefFeatures/image"></soap:address>
+				location="https://localhost:8010/webServiceRefFeatures/image"></soap:address>
```

Comment:
So technically, the port number also needs to change if this is going to be SSL, but since this address gets overridden it doesn't matter. You can remove this change, since it doesn't impact the test.